packages/components/Checkbox/src/Checkbox.settings.ts

```diff
@@ -21,34 +21,36 @@ export const settings: IComposeSettings<ICheckboxType> = [
         flexDirection: 'row',
         minHeight: 14,
         marginTop: 0,
-        position: 'relative'
-      }
+        position: 'relative',
+      },
     },
     checkbox: {
       style: {
-        height: 14,
-        width: 14,
-        marginEnd: 4,
         borderStyle: 'solid',
-        borderWidth: 1
-      }
+        borderWidth: 1,
+        minHeight: 14,
+        minWidth: 14,
+        marginEnd: 4,
+      },
     },
     checkmark: {
       style: {
+        aspectRatio: 1,
         position: 'relative',
         opacity: 0,
         fontSize: 10,
-        marginStart: 2,
-        top: -1
-      }
+        textAlign: 'center',
+        textAlignVertical: 'center',
+        top: -1,
```

Comment:
Why -1?

Reply:
For visual parity I didn't remove it, but I did find it curious.  @Sammy , thoughts / background?

Reply:
With the textAlignVertical field above, you can probably remove the top: -1